languages/csharp/exercises/concept/lists/.meta/Example.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections.Generic;
+
+public static class Languages
```

Comment:
How do you feel about it being a `static` class? I personally don't mind, but I know that I have a tendency to over-static-fy things. I could also imagine it being a non-static class.

Reply:
I can have my arm twisted, but...

our class is in most places a very thin wrapper around a list so it seems more artificial when we are pretending it is some sort of instantiated LanguageCollection than the static version which is more like a platform for testing lists.

Reply:
Okay. Just wanted to check with you :)